fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/GenericResult.java

```diff
@@ -23,6 +24,16 @@
      */
     protected List<T> errors;
 
+    public GenericResult() {
+        isSuccess = true;
+        errors = new ArrayList<T>();
```

Comment:
The reason I would like to keep errors to NULL is that sometimes it would confuse user if there results in a error list with none elements. And the list will be initialized once addError(..) method is invoked like below snippet. Note that I make it a small size by a specific constructor.
```
public void addError(ValidationError error) {
        if (CollectionUtil.isEmpty(errors)) {
            errors = CollectionUtil.createArrayList(Const.INITIAL_CAPACITY);
        }
        errors.add(error);
    }
```